connectors-home/modules/ROOT/pages/introduction/intro-config-oauth2.adoc

```diff
@@ -0,0 +1,189 @@
+= Configuring OAuth 2.0 for Connectors
+
+Many services enable you to use an OAuth 2.0 authentication scheme for increased security when interacting with their APIs. Many connectors support configuring a standard OAuth 2.0 setup.
+
+== Prerequisites
+
+To configure OAuth 2.0 for a connector, you must have access to the following:
+
+* OAuth provider (Okta, Azure AD, Google, PingFederate, and so on)
+* Anypoint Studio
+* Any connector that supports OAuth 2.0
+
+== OAuth 2.0 Configuration Fields
+
+Here are the configuration fields typically available for OAuth 2.0 for connectors:
+
+*OAuth 2.0 Authorization Code:*
+
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Consumer Key | String | OAuth consumer key, as registered with the service provider. | | X
+| Consumer Secret | String | OAuth consumer secret, as registered with the service provider. | | X
+| Authorization URL | String | URL of the service provider's authorization endpoint in which the user is redirected to provide their user credentials. This URL typically ends with `/authorize`. | | X
+| Access Token Url | String | URL of the service provider's access token endpoint in which the application automatically exchanges an authorization code for an access token. | | X
+| Scopes | String | Space-delimited list of OAuth scopes to request during the OAuth dance. This value defaults to the scopes in the annotation. | `offline_access` |
+| Resource Owner ID | String | ID of the owner's user or account authorizing the connector to access their resources. | |
+|===
+
+*OAuth 2.0 Callback Configuration:*
+
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Callback Path | String | Path on your server in which the user's browser redirects to when authorization is successful with the OAuth provider. The callback path is typically `/callback`. This configuration uses your *HTTP Listener* setup to build the callback path, for example, `\http://0.0.0.0:{port}/callback`. | | X
```

Comment:
This is a tough first sentence. Maybe we can talk this one out. 

Server path to which the user's browser is redirected after the user (or is it the app?) successfully authenticates to the OAuth provider.

Reply:
Saying "your server" doesn't sound quite right. Whose server is this - the connector's server, the user's server, or the OAuth provider's server?